src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.html

```diff
@@ -14,4 +14,8 @@
         aria-hidden="true">
     </dot-icon>
 </div>
-<dot-sub-nav [data]="data" (itemClick)="handleItemClick($event)"></dot-sub-nav>
+<dot-sub-nav [data]="data"
+             [class.collapsed]="collapsed"
+             [class.contextmenu]="contextmenu"
```

Comment:
Is simpler to pass the `collapsed` as an `@Input` to `dot-sub-nav` and handle all the styling INSIDE the `dot-sub-nav` which is where it supposed.

Basically the idea is to tell the `dot-sub-nav` that works differently if `collapsed=true`.